src/components/styles/mixins.module.scss

```diff
@@ -49,7 +49,7 @@ $marker-axis-offset: 5px;
   bottom: -5px;
   margin-left: -0.5px;
   box-shadow: 0px 0px white inset, 1px 0px white, -1px 0px white;
-  z-index: 4;
+  z-index: 10;
```

Comment:
I'd throw an extra zero on this - we don't know how people might use their z-index but 10 isn't that many layers to play with